packages/terra-core-docs/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * Changed
   * Updated all paginator examples for `terra-paginator` to use new props.
   * Updated imports from `src` to `lib`.
+  * Updated `terra-form-radio-field` example to display selected value.
```

Comment:
This needs to be moved to the unreleased section.

Reply:
updated : https://github.com/cerner/terra-core/pull/3868/files#diff-0d87855fd9582c564e6ca6a9654fcfa4099b7e717a0704a990d80316f05bdf59